zerver/actions/create_user.py

```diff
@@ -254,7 +254,7 @@ def process_new_human_user(
                 get_system_bot(settings.NOTIFICATION_BOT, prereg_user.referred_by.realm_id),
                 prereg_user.referred_by,
                 _("{user} accepted your invitation to join Zulip!").format(
-                    user=f"{user_profile.full_name} <`{user_profile.email}`>"
+                    user=f"@_**{user_profile.full_name}**"
```

Comment:
Please use the `silent_mention_syntax_for_user` function -- it handles the case of duplicate full names correctly.

Reply:
I have now used `silent_mention_syntax_for_user`